wire-ios/Wire-iOS Tests/ConversationMessageCell/CompleteReactionPickerViewControllerTests.swift

```diff
@@ -36,6 +36,7 @@ final class CompleteReactionPickerViewControllerTests: BaseSnapshotTestCase {
 
     override func tearDown() {
         sut = nil
+        RecentlyUsedEmojiPeristenceCoordinator.store(RecentlyUsedEmojiSection(capacity: 15))
```

Comment:
Does that need to be called in setUp?

Reply:
How about you initialize it in setUp and set it to nil in tearDown? I think that would be more structured regarding the lifecycle of the test?

Reply:
cant pass nil to function and it's static. I don't create instance of RecentlyUsedEmojiPeristenceCoordinator